src/views/project/publishPartition/SummaryView.vue

```diff
@@ -0,0 +1,162 @@
+      and bibliographic citations associated with the characters and taxa will
+      be copied into the new project. The new project will be left in an
+      unpublished state to allow you time to review prior to publication.
+    </p>
+    <p>
+      Note that the new project you create here will be separate from P4091
+      (Test) and create only the subset of information described above. Changes
+      made to the newly created project will not affect P4091 or vice versa.
+    </p>
+    <p>
+      Note that publishing a partition may consume significant server resources,
+      particularly storage. Please be considerate and only publish when you need
+      to.
+    </p>
+
+    <div v-if="!taxa || !characters">
+      <p>
+        You cannot publish this partition because it is empty. Please add
+        characters and taxa and try again.
+      </p>
+    </div>
+
+    <div v-else>
+      <form @submit.prevent="publishPartition">
+        <div v-if="onetimeMedia.length">
+          <p>
+            This partition contains {onetimeMedia.length} media released for
+            onetime use. Due to this license, these media may not be usable in a
+            derivative project. You must either change the media reuse license,
+            or choose to keep the media in the current project, the new parition
+            based project or both (bypassing one-time use).
+          </p>
+          <li v-for="media in onetimeMedia" class="list-group-item">
+            <RouterLink
+              :to="`/myprojects/${projectId}/media/${media.media_id}`"
+            >
+              <button type="button">Onetime Media File</button>
+            </RouterLink>
+          </li>
+          <p>How would you like to handle onetime use media?</p>
+          <div class="list-group-item-name">
+            <label class="item">
+              <input
+                type="radio"
+                class="form-check-input"
+                name="onetimeAction"
+                value="1"
+                :checked="true"
+              />
+              Keep onetime use media in existing project
+            </label>
+          </div>
+          <div class="list-group-item-name">
+            <label class="item">
+              <input
+                type="radio"
+                class="form-check-input"
+                name="onetimeAction"
+                value="100"
+              />
+              Move onetime use media to new project
+            </label>
+          </div>
+          <div class="list-group-item-name">
+            <label class="item">
+              <input
+                type="radio"
+                class="form-check-input"
+                name="onetimeAction"
+                value="1"
+              />
+              Use one-time use media in both existing project and new partition
+              based project
+            </label>
+          </div>
+        </div>
+        <button type="submit">Publish</button>
```

Comment:
The publish button should not be disabled when there are no taxa and characters in the project.

Reply:
And there should also be a Cancel button to go back to the overview page.